R/board_s3.R

```diff
@@ -209,14 +215,15 @@ pin_fetch.pins_board_s3 <- function(board, name, version = NULL, ...) {
 
 #' @export
 pin_store.pins_board_s3 <- function(board, name, paths, metadata,
-                                    versioned = NULL, ...) {
+                                    versioned = NULL, x = NULL, ...) {
```

Comment:
The function signature here needed to be made the same as other `pin_store` methods.